nni/tools/nnictl/config_schema.py

```diff
@@ -383,14 +389,21 @@ def validate(self, data):
 
 frameworkcontroller_config_schema = {
     'frameworkcontrollerConfig': Or({
-        Optional('storage'): setChoice('storage', 'nfs', 'azureStorage'),
+        Optional('storage'): setChoice('storage', 'nfs', 'azureStorage', 'pvc'),
         Optional('serviceAccountName'): setType('serviceAccountName', str),
         'nfs': {
             'server': setType('server', str),
             'path': setType('path', str)
-        }
+        },
+        Optional('namespace'): setType('namespace', str),
+    }, {Optional('configPath'): setType('configPath', str),
+        Optional('storage'): setChoice('storage', 'nfs', 'azureStorage', 'pvc'),
+        Optional('serviceAccountName'): setType('serviceAccountName', str),
+        Optional('configPath'): setType('configPath', str),
+        'pvc': {'path': setType('server', str)},
```

Comment:
`configPath` setting is only for `pvc`?

Reply:
I changed it so configPath is optional for NFS and Azure and mandatory for PVC, as it is required there to attach a PVC at all (via custom template)